geoengine/raster.py

```diff
@@ -213,6 +213,12 @@ def spatial_partition(self) -> gety.SpatialPartition2D:
     def spatial_resolution(self) -> gety.SpatialResolution:
         return self.geo_transform.spatial_resolution()
 
+    def is_empty(self) -> bool:
+        ''' Returns true if the tile is empty'''
```

Comment:
why do we not know if the tile is empty? Does the record batch in `from_ge_record` just contains all no data pixel? In the backend we handle empty tiles differently, so doesn't seem to make sense to me that we materialize them for sending them through the network...

also: I think instead of computing the num_pixels I think you can just say self.data.size (?)

Reply:
yes it makes no sense but this is how we do it at the moment.

Reply:
pyarrow Array has no size as i see it